File: SymfonyApplication/src/Entity/InformationsSupplementaires.php
<?php

namespace App\Entity;

use App\Repository\InformationsSupplementairesRepository;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=InformationsSupplementairesRepository::class)
 */
class InformationsSupplementaires
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=20, nullable=true)
     */
    private $nom;

    /**
     * @ORM\Column(type="string", length=20, nullable=true)
     */
    private $prenom;

    /**
     * @ORM\Column(type="string", length=20, nullable=true)
     */
    private $tell;

    /**
     * @ORM\Column(type="string", length=50, nullable=true)
     */
    private $image;

    /**
     * @ORM\OneToOne(targetEntity=User::class, inversedBy="informationsSupplementaires", cascade={"persist", "remove"})
     * @ORM\JoinColumn(nullable=false)
     */
    private $User;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(?string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(?string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getTell(): ?string
    {
        return $this->tell;
    }

    public function setTell(?string $tell): self
    {
        $this->tell = $tell;

        return $this;
    }

    public function getImage(): ?string
    {
        return $this->image;
    }

    public function setImage(?string $image): self
    {
        $this->image = $image;

        return $this;
    }

    public function getUser(): ?User
    {
        return $this->User;
    }

    public function setUser(User $User): self
    {
        $this->User = $User;

        return $this;
    }
}

File: JavaApplication/src/HolidaysHiatus/entites/InformationsSupplementaires.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package HolidaysHiatus.entites;

/**
 *
 * @author ASUS
 */
public class InformationsSupplementaires {
    int id;
    int user_id;
    String nom;
    String prenom;
    String tell;
    String image;

    public InformationsSupplementaires(int id, int user_id, String nom, String prenom, String tell, String image) {
        this.id = id;
        this.user_id = user_id;
        this.nom = nom;
        this.prenom = prenom;
        this.tell = tell;
        this.image = image;
    }

    public InformationsSupplementaires(int user_id, String nom, String prenom, String tell) {
        this.user_id = user_id;
        this.nom = nom;
        this.prenom = prenom;
        this.tell = tell;
    }
    

    public InformationsSupplementaires(int user_id, String nom, String prenom, String tell, String image) {
        this.user_id = user_id;
        this.nom = nom;
        this.prenom = prenom;
        this.tell = tell;
        this.image = image;
    }

    public InformationsSupplementaires() {
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getUser_id() {
        return user_id;
    }

    public void setUser_id(int user_id) {
        this.user_id = user_id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getTell() {
        return tell;
    }

    public void setTell(String tell) {
        this.tell = tell;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public InformationsSupplementaires(int user_id) {
        this.user_id = user_id;
    }
    
    
    
}
